src/cdn_definitions/_impl/__init__.py

```diff
@@ -31,6 +83,10 @@ class PathAlias(object):
     used to make two directory trees on CDN serve identical content."""
 
     def __init__(self, **kwargs):
+        warnings.warn(
+            "Global PathAlias is deprecated - please use global load_data instead",
```

Comment:
Maybe we can drop the 'global' term, I had mentioned that before since we were going to have a CdnData class and it looked like calling e.g. rhui_alias on an instance of that class, rather than the global one, would still be supported.

Now as we've dropped the CdnData class concept and there's no non-global form of these APIs, we could just delete the word "global" here.